packages/scandipwa/src/component/CategoryConfigurableAttributes/CategoryConfigurableAttributes.component.js

```diff
@@ -88,6 +89,30 @@ export class CategoryConfigurableAttributes extends ProductConfigurableAttribute
         );
     }
 
+    renderConfigurableAttributeValue(attribute) {
+        const {
+            getIsConfigurableAttributeAvailable,
+            handleOptionClick,
+            getLink,
+            isSelected,
+            show_product_count
+        } = this.props;
+
+        const { attribute_value } = attribute;
+
+        return (
+            <ProductAttributeValue
+              key={ attribute_value }
+              attribute={ attribute }
+              isSelected={ isSelected(attribute) }
+              isAvailable={ getIsConfigurableAttributeAvailable(attribute) }
+              onClick={ handleOptionClick }
+              getLink={ getLink }
+              showProductCount={ show_product_count }
```

Comment:
Why can not we just get this value from the `ProductAttributeValue` directly?

Reply:
```ProductAttributeValue``` is used within multiple places and not only in layered navigation, if we are getting configuration directly then product count will also be displayed in PDP description area and configuration area, by setting default to false and passing its value from ```CategoryConfigurableAttributes``` I can achieve that count is only displayed in layered navigation